ports/supermethanebros/Super Methane Brothers.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+GAMEDIR="/$directory/ports/supermethanebros"
+
+export LD_LIBRARY_PATH="$GAMEDIR/libs:$LD_LIBRARY_PATH"
```

Comment:
```suggestion
export LD_LIBRARY_PATH="$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"
```
Also rename `libs` to `libs.aarch64`

Reply:
I get this, but I wasn't planning on doing anything other than the aarch64 build so unless that changes this won't matter.  I'll also need to rename the binary to follow the regimen.  I can do this, but since I wasn't planning on doing other architectures...  unless you'd like me to.

Reply:
Let's ask @Cebion 😅 Can we skip the .aarch64 or is it something we want to enforce for all ports ?

Reply:
If a port gets multiarch it's easier. So i just do the aarch64 folder and the libs part. Doesn#t hurt and makes it easier in the long run for other potential ports. @cdeletre @Slayer366

Reply:
I've made the adjustments and I briefly retested to ensure I didn't fat-finger anything in the shell script.